### ISSUE TITLE
Create new command rit "clean repo local" or "delete repo local"

### ISSUE THREAD

**Issue by fabianofernandeszup** [ZupIT/ritchie-cli#475]
**What would you like to add**:
Command to clean the local formulas folder.

**Why this is necessary**:
When the user clones the repository to improve a formula, he build it locally to perform your tests.
As soon as he finishes this, commit the formula and generate a new release, he is unable to use the repo formula because local formulas have priority.
Then this command will be to clean the local formula repo.

**Comment by brunonmelo** [comment]
I'm currently work on this issue.

**Comment by GuillaumeFalourd** [comment]
Ok. I've assigned it to you @brunonmelo 
Thank you for contributing ! 👍

**Comment by brunonmelo** [comment]
I created a new Pull request for this issue. #597 
I'm still need to fix the unit test, so I marked as WIP.

There is a important consideration on the pull request:
I noticed that after you delete your local repo, Ritcher will not build or run your local formula, even if already exists on your workspace. The user will need to execute the command rit build formula to this formula be available again.
It would be greate to have some command that build all the formulas of a repository all at once.
